js/packages/teams-ai/src/MessageExtensions.spec.ts

```diff
@@ -27,7 +28,8 @@ describe('MessageExtensions', () => {
     const adapter = new TestAdapter();
     let mockApp: Application;
     beforeEach(() => {
-        mockApp = new Application({ adapter });
+        mockApp = new Application();
+        sinon.stub(mockApp, 'adapter').get(() => adapter);
```

Comment:
This actually needs a sandbox (Learned this an hour ago)

```
    let sinonSandbox: sinon.SinonSandbox;
    let createStub: sinon.SinonStub;

// ----

beforeEach(() => {
        sinonSandbox = createSandbox();
        createStub = sinonSandbox.stub(axios, 'create').returns(mockAxios);
});
// ...
    afterEach(() => {
        sinonSandbox.restore();
    });

// test: 
 assert.equal(createStub.called, true);
```

I think there are a number of places this needs to be fixed, not just here, so if you would prefer to file an issue and assign it to me I can make the fixes all at once. :)